core/src/apps/cardano/layout.py

```diff
@@ -76,6 +90,55 @@ async def confirm_sending(ctx: wire.Context, amount: int, to: str) -> None:
     await require_confirm(ctx, Paginated(pages))
 
 
+async def confirm_sending_token_group(
+    ctx: wire.Context, token_group: CardanoAssetGroupType
+) -> None:
+    page1 = Text("Confirm transaction", ui.ICON_SEND, ui.GREEN)
+    page1.bold("Policy id: ")
+    page1.mono(hexlify(token_group.policy_id).decode())
+    await require_confirm(ctx, page1)
+
+    for idx, token in enumerate(token_group.tokens):
```

Comment:
`enumerate(token_group.tokens, 1)` and you won't need to use `token_index + 1`

Reply:
of course then it would be better to call it `token_number`, since "index" seems to imply 0-based